audio_context.html

```diff
@@ -0,0 +1,321 @@
+  }
+
+  h1 {
+    text-align: center;
+  }
+
+  main {
+    outline: thick double;
+    background-color: whitesmoke;
+  }
+
+  fieldset {
+    display: flex;
+    border: none;
+    background: yellow;
+  }
+
+  form > fieldset {
+    flex-direction: var(--flex-direction);
+  }
+
+  fieldset fieldset {
+    background: red;
+  }
+
+  fieldset input[type="range"] {
+    width: 20vw;
+    display: inline-block;
+    top: 0.3em;
+    position: relative;
+  }
+
+  form > fieldset > label {
+    background: turquoise;
+    line-height: var(--line-height);
+  }
+
+  form > header {
+    background: yellow;
+  }
+
+  [type="checkbox"] {
+  }
+
+</style>
+
+</head>
+
+<body>
+
+  <header>
+    <h1>DevPunks Sequencer</h1>
+  </header>
+
+<main>
+
+  <form action=#>
+
+  <header>
+    DevPunks Sequencer
+
+    <label>
+      BPM
+      <input type=range min=10 max=522 value=251>
+    </label>
+
+    <label>
+      Play
+      <input name="Play" type="checkbox">
+    </label>
+  </header>
+
+  <fieldset>
+
+    <legend>Sweep</legend>
+
+    <label>
+      Attack
+      <input type="range" name="attack" min="0" max="1" value="0.2" step="0.1">
+    </label>
+
+    <label>
+      Release
+      <input type="range" name="release" min="0" max="1" value="0.5" step="0.1">
+    </label>
+
+    <fieldset>
+
+      <legend hidden>Checkboxes</legend>
+
+      <label>
+        <input name=pads type="checkbox">
+      </label>
+
+      <label>
+        <input name=pads type="checkbox">
+      </label>
+
+      <label>
+        <input name=pads type="checkbox">
+      </label>
+
+      <label>
+        <input name=pads type="checkbox">
+      </label>
+
+    </fieldset>
+
+  </fieldset>
+
+  <fieldset>
+
+    <legend>Pulse</legend>
+
+    <label>
+      Hz
+      <input type="range" name="hertz" min="660" max="1320" value="880" step="1">
+    </label>
+
+    <label>
+      LFO
+      <input type="range" name="lfo" min="20" max="40" value="30" step="1">
+    </label>
+
+  <fieldset>
+
+    <legend hidden>Checkboxes</legend>
+
+    <label>
+      <input name=pads type="checkbox">
+    </label>
+
+    <label>
+      <input name=pads type="checkbox">
+    </label>
+
+    <label>
+      <input name=pads type="checkbox">
+    </label>
+
+    <label>
+      <input name=pads type="checkbox">
+    </label>
+
+  </fieldset>
+
+  </fieldset>
+
+  <fieldset>
+    <legend>Noise</legend>
+
+    <label>
+      Duration
+    <input type="range" name="duration" min="0" max="2" value="1" step="0.1">
+    </label>
+
+    <label>
+      Band
+    <input type="range" name="band" min="400" max="1200" value="1000" step="5">
+    </label>
+
+    <fieldset>
+
+    <legend hidden>Checkboxes</legend>
+
+    <label>
+      <input name=pads type="checkbox">
+    </label>
+
+    <label>
+      <input name=pads type="checkbox">
+    </label>
+
+    <label>
+      <input name=pads type="checkbox">
+    </label>
+
+    <label>
+      <input name=pads type="checkbox">
+    </label>
+
+  </fieldset>
```

Comment:
Triple closing `</fieldset>`s on the same nesting level? 🤔

Reply:
It should be one <fieldset> with 4 nesting <fieldset>s withing , might have created to many spaces between tags,could be confusing.